backend/config/settings.py

```diff
@@ -302,6 +303,15 @@
                 f"https://{AWS_S3_PRIVATE_CUSTOM_DOMAIN}/{AWS_PRIVATE_LOCATION}/"
             )
 
+            # Used for backing up the database https://django-dbbackup.readthedocs.io/en/master/storage.html#id2
+            DBBACKUP_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+            DBBACKUP_STORAGE_OPTIONS = {
```

Comment:
Yea I think that would be the right approach. As of this current stage, we just needed somewhere to drop the dumps, since we don't yet have that dedicated bucket. @mogul we have some time before this needs to get merged, do you want to sync and get a meta maintained bucket available?